examples/cve_published_last_month_export_xlsx.py

```diff
@@ -0,0 +1,122 @@
+"""Export vulnerabilities published last month"""
```

Comment:
If you filter only the exploitable ones specify it here, or don't filter.

Reply:
got rid of the filter and updated the docstring to "active vulnerabilities"